wp-content/civi-extensions/goonjcustom/Civi/CollectionCampVolunteerFeedbackService.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Civi;
+
+use Civi\Core\Service\AutoSubscriber;
+
+/**
+ * Collection Camp Outcome Service.
+ */
+class CollectionCampVolunteerFeedbackService extends AutoSubscriber {
+
+  /**
+   * Define subscribed events.
+   */
+  public static function getSubscribedEvents() {
+    return [];
+  }
+
+  /**
+   * Send the feedback reminder email to the volunteer.
+   *
+   * @param int $volunteerEmailId
+   * @param string $from
+   * @param string $campAddress
+   * @param int $collectionCampId
+   * @param \DateTime $endDate
+   */
+  public static function sendVolunteerFeedbackReminderEmail($volunteerEmailId, $from, $campAddress, $collectionCampId, $endDate, $volunteerName) {
+    $mailParams = [
+      'subject' => 'Reminder to share your feedback for ' . $campAddress . ' on ' . $endDate->format('Y-m-d'),
+      'from' => $from,
+      'toEmail' => $volunteerEmailId,
+      'replyTo' => $from,
+      'html' => self::getVolunteerFeedbackReminderEmailHtml($volunteerName, $collectionCampId, $volunteerEmailId),
+    ];
+
+    $emailSendResult = \CRM_Utils_Mail::send($mailParams);
+
+    if (!$emailSendResult) {
+      \Civi::log()->error('Failed to send feedback reminder email', [
+        'volunteerEmailId' => $volunteerEmailId,
+        'volunteerEmail' => $volunteerEmail,
+      ]);
+      throw new \CRM_Core_Exception('Failed to send feedback reminder email');
+    }
+  }
+
+  /**
+   * Generates HTML content for the volunteer feedback reminder email.
+   *
+   * @param string $volunteerName
+   * @param int $collectionCampId
+   * @param int $volunteerEmailId
+   *
+   * @return string
+   */
+  public static function getVolunteerFeedbackReminderEmailHtml($volunteerName, $collectionCampId, $volunteerEmailId) {
+    $homeUrl = \CRM_Utils_System::baseCMSURL();
+    $feedbackFormUrl = $homeUrl . 'volunteer-camp-feedback/#?Eck_Collection_Camp1=' . $collectionCampId;
```

Comment:
This is intentional. This is a special way to send some value via in CiviCRM's Formbuilder. So if you notice the same thing and your comment should to ask whether we are using the CiviCRM Formbuilder for the URL if not then it might be a mistake.